content/en/docs/concepts/security/security-checklist.md

```diff
@@ -159,4 +159,48 @@ Production.
 
 The blog post
 [A Closer Look at NSA/CISA Kubernetes Hardening Guidance](/blog/2021/10/05/nsa-cisa-kubernetes-hardening-guidance/#building-secure-container-images)
-provides more guidance on how to build secure container images.
+## Admission Controllers
+
+The following are a number of admission controllers that could be considered to
+enhance the security posture of your application. This list includes
+controllers that may be referenced in other parts of this document:
+
```

Comment:
Style tip: avoid “ASCII art”; use HTML or Markdown to imply structure.

Try a [definition list](https://www.markdownguide.org/extended-syntax/#definition-lists).

Reply:
Ah, I hadn't worked out that intent there.
I would still use a definition list, mind.

Reply:
You are right I will convert it then, thanks a lot!! :)

Reply:
This is way better thanks! https://github.com/kubernetes/website/pull/33992/commits/7748826f85e8170b1f25be1251d155812f7ac507